README.md

```diff
@@ -171,48 +176,43 @@ This API looks for a key. If the key does not exist, it adds a new Serialized ke
 As explained above, when constructing off-heap OakMap, the memory capacity (per OakMap instance) needs to be specified. OakMap allocates the off-heap memory with the requested capacity at construction time, and later manages this memory.
 This memory (the entire given capacity) needs to be released later, thus OakMap implements AutoClosable. Be sure to use it within try-statement or better invoke OakMap's close() method when OakMap is no longer in use.
 
-Please pay attention that multiple views can be defined to the same underlying memory of OakMap. That is when other sub-maps and views (OakBufferView/OakTransformView) are created. Do not worry, the true memory release will happen only when last of those views is closed.
-However, note that each sub-map is in particular an OakMap and thus AutoCloseable and needs to be closed (explicitly or implicitly). Similarly, OakBufferView and OakTransformView are AutoCloseable objects and need to be closed. Again, close() can be invoked on different objects referring to the same underlying memory, but the final release will happen only once.
+Please pay attention that multiple views can be defined to the same underlying memory of OakMap. That is when other sub-maps are created. Do not worry, the true memory release will happen only when last of those views is closed.
```

Comment:
Did you mean ZeroCopyMaps instead of views? There are no views now...

Reply:
> That is when other sub-maps are created.

Views here mean sub maps. Perhaps this should be more clearly indicated. I will change the wording.